feature-group-definitions/details-name.yml

```diff
@@ -0,0 +1,5 @@
+name: Mutually exclusive <details> elements
```

Comment:
Huh, I didn't know about this until today.

Reply:
It's new!